quizcon/main/tests/test_models.py

```diff
@@ -3,13 +3,60 @@
     CourseTestMixin, QuizFactory, QuestionFactory, QuestionResponseFactory,
     QuizSubmissionFactory
 )
+from quizcon.main.models import Quiz
 
 
 class BasicModelTest(TestCase):
     def test_dummy(self):
         self.assertTrue(True)
 
 
+class QuizCloneTest(CourseTestMixin, TestCase):
+    def setUp(self):
+        self.setup_course()
+
+    def test_clone(self):
+        q = QuizFactory(course=self.course, title='cloned quiz')
+        q.question = QuestionFactory(quiz=q)
+
+        cloned_pk = q.clone().pk
```

Comment:
This is a small nit, `q.clone()` returns the cloned quiz. I'm not sure that you need to need to call `get`